gslb/gslbutils/gdputils.go

```diff
@@ -269,6 +306,20 @@ func (gf *GlobalFilter) ComputeChecksum() {
 	for _, ts := range gf.TrafficSplit {
 		cksum += utils.Hash(ts.ClusterName + strconv.Itoa(int(ts.Weight)))
 	}
+	if gf.SitePersistence.Enabled {
```

Comment:
I think we can get away with just using one field for persistence profile and not require a specific boolean.

Reply:
Yeah, makes sense. Will remove the enabled field.